stock_release_channel/views/stock_release_channel_views.xml

```diff
@@ -264,7 +341,7 @@
                                                 name="count_picking_done"
                                                 widget="progressbar"
                                                 title="Overall Progress"
-                                                options="{'show_fractional': true, 'current_value': 'count_picking_done', 'max_value': 'count_picking_full_progress', 'editable': false}"
+                                                options="{'show_fractional': True, 'current_value': 'count_picking_done', 'max_value': 'count_picking_full_progress', 'editable': false}"
```

Comment:
is this intended?

Reply:
looks correct to me

Reply:
The previous version should work as expected no? (`options` is a JSON parameter normally)